ios/Exponent/Supporting/Info.plist

```diff
@@ -45,7 +45,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>2.8.4</string>
+	<string>2.8.4.103059</string>
```

Comment:
This is an incidental change probably caused by a failed build where the `clear_bundle_version` task never ran. Don't want/need to commit it, otherwise we'll just have git noise when a future commit removes it again.